modules/protocol/resources/META-INF/smithy/smithy4s.meta.smithy

```diff
@@ -11,6 +11,9 @@ metadata suppressions = [
 
 namespace smithy4s.meta
 
+@trait(selector: ":is(service, operation)")
+structure only {}
```

Comment:
**nitpick:** if we're sticking to "just operations", I'd call it sth like `keepOperation`. I'm ok with the current name though

Reply:
We may expand it later on with additional uses, so I'm happy with the name.